massa-models/src/operation.rs

```diff
@@ -695,14 +722,18 @@ impl Deserializer<OperationType> for OperationTypeDeserializer {
                         context("Failed data deserialization", |input| {
                             self.data_deserializer.deserialize(input)
                         }),
+                        context("Failed data deserialization", |input| {
```

Comment:
The message here is the same as above. It should be an other to be able to find easily the source of the error message.

Reply:
Just fixed the error message :)